cli/src/commands/explore/mod.rs

```diff
@@ -0,0 +1,147 @@
+use crate::utils::{print_docs_with_indent, FileOrUrl};
+use clap::{Parser as ClapParser, Subcommand};
+
+use std::fmt::Write;
+
+use std::write;
+
+use codec::Decode;
+use color_eyre::eyre::eyre;
+use frame_metadata::v15::RuntimeMetadataV15;
+use frame_metadata::RuntimeMetadataPrefixed;
+
+use syn::__private::str;
+
+use crate::commands::explore::calls::{explore_calls, CallsSubcommand};
+use crate::commands::explore::constants::{explore_constants, ConstantsSubcommand};
+use crate::commands::explore::storage::{explore_storage, StorageSubcommand};
+
+use subxt::Metadata;
+
+mod calls;
+mod constants;
+mod storage;
+
+/// Explore pallets, calls, call parameters, storage entries and constants. Also allows for creating (unsigned) extrinsics.
+///
+/// # Example
+///
+/// ## Pallets
+///
+/// Show the pallets that are available:
+/// ```
```

Comment:
Nit: All doc examples with "```" are assumed to be rust doc comments; I'd do "```text".

I guess rustdoc isn't trying to interpret these or something because it's a binary and not library project; I'd have expected to see CI others ordinarily as it tried to run them.

I don't expect anybody will ever see these comments, so they are only really useful to us.